vault/changelog.release.2022-08-09.md

```diff
@@ -0,0 +1,71 @@
+---
+id: 482bl67lvjd0r8ht0cngxmk
+title: 0.107
+desc: ''
+updated: 1659972909881
+created: 1659968708044
+---
+
+Dendron 0.107 has sprouted  🌱
+August 09, 2022
+
+**Add Toggle Preview Lock command**: You can now lock in the preview of a specific note and navigate other notes without changing your preview.
```

Comment:
update wording:

**Preview Locking**: You can now lock the preview to a specific note which will remain pinned even as you navigate to different notes.